src/common/entryPointHandler.ts

```diff
@@ -11,34 +16,37 @@ export class EntryPointHandler {
     private outputChannel: OutputChannel;
 
     constructor(outputChannel?: OutputChannel) {
```

Comment:
If we pass the logger directly here to the constructor instead of the `outputChannel`, we could remove the "vscode" import from this common class.